redfish-core/lib/network_protocol.hpp

```diff
@@ -152,7 +165,7 @@ inline void getNetworkData(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
         boost::beast::http::field::link,
         "</redfish/v1/JsonSchemas/ManagerNetworkProtocol/NetworkProtocol.json>; rel=describedby");
     asyncResp->res.jsonValue["@odata.type"] =
-        "#ManagerNetworkProtocol.v1_5_0.ManagerNetworkProtocol";
+        "#ManagerNetworkProtocol.v1_9_0.ManagerNetworkProtocol";
```

Comment:
Do we have this version of the schema pulled ?

Reply:
yes, we have ManagerNetworkProtocol.v1_10 also in upstream
but in downstream 1060 has only ManagerNetworkProtocol.v1_9